source/backups.rst

```diff
@@ -271,7 +315,73 @@ Following steps are needed to restore a previously saved backup:
                endpointURL: https://URL-OF-THE-S3-COMPATIBLE-STORAGE
            ...
 
-   After that, the actual restoration process can be started as follows:
+2. After that, the actual restoration process can be started as follows:
+
+   .. code:: bash
+
+      kubectl apply -f deploy/backup/restore.yaml
+
+.. note:: Storing backup settings in a separate file can be replaced by passing
+   its content to the ``kubectl apply`` command as follows:
+
+   .. code:: bash
+
+      cat <<EOF | kubectl apply -f-
+      apiVersion: "pxc.percona.com/v1"
+      kind: "PerconaXtraDBClusterRestore"
+      metadata:
+        name: "restore1"
+      spec:
+        pxcCluster: "cluster1"
+        backupName: "backup1"
+      EOF
+
+.. _backups-pitr-restore:
+
+Restoring backup with point-in-time recovery
+********************************************
+
+.. note:: Disable the point-in-time functionality on the existing cluster before
+          restoring a backup on it, regardless of whether the backup was made
+          with point-in-time recovery or without it.
+
+If the point-in-time recovery feature :ref:`was enabled<backups-pitr-binlog>`,
+you can put additional restoration parameters to the ``restore.yaml`` file
+``pitr`` section for the most fine-grained restoration.
+
+* ``backupSource`` key should contain ``destination`` key equal to the s3 bucket
+  with a special ``s3://`` prefix, followed by the necessary S3 configuration
+  keys, same as in ``deploy/cr.yaml`` file: ``s3://S3-BUCKET-NAME/BACKUP-NAME``,
+* ``type`` key can be equal to one of the following options,
+  * ``date`` - roll back to specific date,
+  * ``skip`` - roll back, skip the transaction, roll over,
+  * ``latest`` - recover to the latest possible transaction,
+* ``date`` key is used with ``type=date`` option - it contains value in
+  datetime format,
+* ``gtidSet`` key is used with ``type=skip`` option - it contain exact GTID.
+
+The resulting ``restore.yaml`` file may look as follows:
+
+.. code-block:: yaml
+
+   apiVersion: pxc.percona.com/v1
+   kind: PerconaXtraDBClusterRestore
+   metadata:
+     name: restore1
+   spec:
+     pxcCluster: cluster1
+     backupName: backup1
+     pitr:
+       type: date
+       date: "2020-12-31 09:37:13"
+       backupSource:
+         destination: s3://S3-BUCKET-NAME/BACKUP-NAME
```

Comment:
I don't think `destination` is used for `pitr` part of restore.yaml file, but `bucket` is mandatory, but it goes under `pitr->backupSource->s3->bucket`
something like this:
```
  pitr:
    type: latest
    backupSource:
      s3:
        bucket: tomislav-backup/binlogs
        credentialsSecret: my-cluster-name-backup-s3
        endpointUrl: https://s3.us-west-2.amazonaws.com
        region: us-west-2
```